iOS/G3MiOSSDK/Commons/Basic/ShortBufferElevationData.cpp

```diff
@@ -37,10 +37,49 @@ _buffer(buffer)
       break;
     }
   }
+    
+
+  _max = IMathUtils::instance()->minInt16();
+  _min = IMathUtils::instance()->maxInt16();
+  _children = IMathUtils::instance()->minInt16();
+  _similarity = IMathUtils::instance()->minInt16();
+}
+
+ShortBufferElevationData::ShortBufferElevationData(const Sector& sector,
+                                                   const Vector2I& extent,
+                                                   const Sector& realSector,
+                                                   const Vector2I& realExtent,
+                                                   short* buffer,
+                                                   int bufferSize,
+                                                   double deltaHeight,
+                                                   short max,
+                                                   short min,
+                                                   short children,
+                                                   short similarity) :
+BufferElevationData(sector, extent, realSector, realExtent, bufferSize, deltaHeight),
```

Comment:
We have to talk with @amazingsmash about the pending refactor of ElevationData stuff, probably the right place to put this "extra" information is there (some ElevationData subclass).